Purchases/Logging/Strings/PurchaseStrings.swift

```diff
@@ -196,6 +197,8 @@ extension PurchaseStrings: CustomStringConvertible {
                 " for refund."
         case .begin_refund_no_active_entitlement:
             return "Could not begin refund request. No active entitlement."
+        case .begin_refund_multiple_active_entitlements:
+            return "Could not begin refund request. Multiple active entitlements."
```

Comment:
🤔 Maybe here we could actually point folks to use the method that allows you to specify the entitlement? 
As in: 

```swift
Could not begin refund request. There are multiple active entitlements. Use 
`beginRefundRequest(forEntitlement:)` to specify a single entitlement instead.
```

I realize that it's an error you can't really surface to the user, though. I'm hoping that maybe it'd get caught _before_ getting to real users